sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -652,7 +665,10 @@ def test_check_estimator():
         r"support \S{3}_64 matrix, and is not failing gracefully.*"
```

Comment:
I was trying this today (imitating the same logic as in `LargeSparseNotSupportedClassifier`), but I'm not sure if successfully. I will push it in a moment, please have a look. 

I don't know how I can see myself where the checks would fail. When I use the `test_estimator_checks.py`, the errors are immediately captured. I was expecting some different behavior that I could see in the debugger, like jumping right back to the `raise` statement in the check and finishing that check then. But it looked alike anyways.